src/pages/home-page/HomePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { Event, Store, createEvent, createStore } from 'effector';
+import { useStore } from 'effector-react';
+import React from 'react';
+import { Button, Container, Paper, TextField } from 'shared/ui';
+
+const setUsername: Event<string> = createEvent();
+const setPassword: Event<string> = createEvent();
+
+const $usernameStore: Store<string> = createStore('').on(setUsername, (_, value) => value);
```

Comment:
okay, let's consider this data will belong to a part of business logic that we will f.e send to the backend to login user, and now you have it on the UI. By FSD all the logic that does not belong to a UI logic (f.e handleChange which is expanding and collapsing the accordion) should stay in the model

so basically you will have a structure like this:

model ----> store.ts | index.ts
ui ---------> conponent.tsx | index.ts
index.ts

inside the store.ts you can have your molecule state. Effector is good enough cause it can be used for a global state as well as for let's say molecule state

inside the store.ts you can create your own domain, for which you will add events, effects and samples that you need